### DISCUSSION TITLE
Seeking antibody recommendations for typing mesangial, acinar, and epsilon cells in human tissues

### DISCUSSION THREAD

**Discussion by radtkea** [IBEXImagingCommunity/ibex_imaging_knowledge_base#43] [Q&A] [answered]
Hello Community!

My colleagues and I are interested in antibodies validated for human fixed frozen or FFPE tissues for phenotyping mesangial cells in the kidney and acinar and epsilon cells in the pancreas. We are open to any suggestions for markers and, most importantly, antibodies for multiplexed imaging. 

For FFPE tissues, we prefer a dual antigen retrieval protocol (pH 6 and pH 9) originally pioneered by the GE Cell DIVE team. 

Thank you for your help,
Andrea

**Comment by anita-gola** [comment, marked as answer]
Hi Andrea, 
While I do not have any expertise in imaging the pancreas, I did find this article which maybe you will find useful? 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3679653/
Wish you the best of luck!

**Reply by radtkea** [reply]
Hi Anita!

Thanks very much for sending along this paper on how to perform histology on pancreatic neoplasms. I appreciate your help! I didn't find any markers for epsilon or acinar cells in this manuscript. 

I will keep you posted.

Best of luck to you as well!
A

**Reply by jfisherCST** [reply]
Hi Andrea, 
Some potential markers of interest for phenotyping mesangial cells in the kidney and acinar and epsilon cells in the pancreas are listed below. Also below are some publications of potential interest.  

Mesangial - PDGFRB
acinar α-Amylase
epsilon cells - Ghrelin
Cell Signaling Technology does have some imaging approved clones for these targets. 
α-Amylase (D55H10) XP® Rabbit mAb #3796 - tested on FFPE human and rat pancreas
PDGF Receptor β (28E1) Rabbit mAb #3169 - tested in FFPE human colon 
Ghrelin (D4K1W) Rabbit mAb #31865 - mouse and rat reactive and tested in mouse stomach on fixed frozen tissue. A note for antibody specificity "Ghrelin (D4K1W) Rabbit mAb recognizes endogenous levels of total ghrelin protein. The antibody detects acylated (n-octanoyl) and non-acylated (desacyl) forms of ghrelin (confirmed by Western blot analysis of synthetic ghrelin protein)."

https://pubmed.ncbi.nlm.nih.gov/33837218/
https://www.nature.com/articles/s41598-019-40481-1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6514973/

Hope this helps. 
Thanks
Jeremy

**Reply by radtkea** [reply]
Hi Jeremy!

Wow, thank you VERY much for writing such a detailed and thoughtful reply. These are great recommendations and resources. I will try these antibodies and post our results on the knowledge-base.

Thank you again,
Andrea

**Reply by mcampthomp** [reply]
Hi Andrea,
For acinar cells in human pancreas, we have used these two: 
Amylase (AMY1A, 2A, 2B), Mouse anti-all [G-10]- Santa Cruz, sc-46657 for 3D (VISIKOL/iDISCO) and 2D FFPE following citrate or BORG AR.
Amylase (AMY2A), Rabbit anti-hu ms dog, RRID:AB_258380 - Sigma A8273-1VL for 3D organoid staining (VISIKOL/iDISCO), Example in https://doi.org/10.3389/fendo.2021.644826

Have not seen a validated xGhrelin yet.

Sincerely,
Martha

**Comment by radtkea** [comment]
Hi Martha!

So great to see you here! Full disclosure: I asked this question on behalf of our other HuBMAP colleagues. The details you provided are tremendously helpful, e.g., 2D or 3D imaging, clearing agent (if applicable), and antigen retrieval conditions. Thank you!

We will look into the antibodies you kindly suggested. I will also report our results with the Ghrelin antibody kindly suggested by Jeremy (CST #31865). Just need a little breathing room for antibody validation fun.

Wow, your frontiers article is a tour de force! Beautiful images and very well written. Loved seeing the table of successful and not successful antibodies along with a special focus on the lymphatics of the pancreas. Let us know if we can help support/showcase your work.

Thank you very much!
A

**Comment by mcampthomp** [comment]
I LOVE your table and was looking for a way to download or leave a comment to end up on github to see your question. Yes would love to find out how the Ghrelin works.

Best website yet- mega congratulations.

Martha

**Reply by radtkea** [reply]
Hi Martha:)

Thanks very much for your kind words! So glad you like it and find it useful. It reflects a decade of imaging in the Germain lab and 1.5 years of tinkering with Dr. Ziv Yaniv with additions from colleagues around the world. Our work with HuBMAP on antibody validation and multiplexed imaging has been critical here too.

Another great question! Ziv designed the whole site as a "downloadable dataset" from Zenodo. You can obtain the reagent resources table by downloading the file on Zenodo (https://zenodo.org/record/7693279#.ZDbglnbMIuU). The file used to generate the table is called "reagent_resources.csv" and is located in the data folder. It will be updated with each release.

We will keep you posted about Ghrelin. We like feeding the knowledge-base:)

Have a lovely day,
A

Thanks again and have a great day,
A
